metecho/api/serializers.py

```diff
@@ -350,6 +398,7 @@ class Meta:
             "origin_sha",
             "branch_diff_url",
             "pr_url",
+            "issue",
```

Comment:
It's not ideal, but I think all of these `serializer` changes should make their way into the relevant `docs/` files too.

Reply:
Fixed in 30c74071cb041faa423eac9b35bbb066d05a4a5f. Also went back and documented previous changes.